jest.config.ts

```diff
@@ -0,0 +1,225 @@
+/**
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+import type { Config } from 'jest';
+import nextJest from 'next/jest.js';
+
+const createJestConfig = nextJest({
+  // Provide the path to your Next.js app to load next.config.js and .env files in your test environment
+  dir: './',
+});
+
+const config: Config = {
+  // All imported modules in your tests should be mocked automatically
```

Comment:
why do we want lots of commented out config in this file? makes it harder to see what config we are actually specifying